examples/Demo/Shared/Pages/TextField/Examples/TextFieldTypes.razor

```diff
@@ -13,6 +13,15 @@
 <h4>Text with InputMode</h4>
 <FluentTextField @bind-Value=value5 TextFieldType="TextFieldType.Text" InputMode="InputMode.Numeric">Text with InputMode</FluentTextField>
 
+<h4>Text with color picker</h4>
+<FluentTextField @bind-Value=value6 TextFieldType="TextFieldType.Color">Html color picker</FluentTextField>
```

Comment:
I think you need to add a `width: 50px` style attribute to display the selected color.
If you don't, the result will look odd :-)

Reply:
![image](https://github.com/user-attachments/assets/5568b2d8-102e-4e98-b3e6-022dacad44ac)
this is with `width:178px`